mirrord/cli/src/main.rs

```diff
@@ -321,6 +321,10 @@ async fn exec(args: &ExecArgs, watch: drain::Watch) -> Result<()> {
         args.binary, args.binary_args
     );
 
+    if args.binary.contains("docker") {
```

Comment:
```suggestion
    if args.binary.contains("docker") || args.binary.contains("podman") || args.binary.contains("nerdctl") {
```

Reply:
Added it as a regex, so we can just add `|whatever`, wdyt? (also is the regex ok? it'll match on things like `foo-docker`)